t/page/house_wikidata.rb

```diff
@@ -0,0 +1,22 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/house_wikidata'
+
+describe 'HouseWikidata' do
+  subject do
+    Page::HouseWikidata.new('austria', 'Nationalrat')
+  end
+
+  it 'should return a house' do
+    subject.house[:name].must_equal 'Nationalrat'
+  end
+
+  it 'should return a page_title' do
+    subject.page_title.must_equal 'EveryPolitician: Austria — Nationalrat'
+  end
+
+  it 'should return the country popolo' do
+    popolo = subject.popolo
+    keys = %i(posts persons organizations meta memberships events areas)
+    (popolo.keys & keys).must_equal keys
```

Comment:
I find this quite confusing. What are you actually testing here? Isn't this just a well disguised version of `popolo.keys.must_equal keys`? 

Aside from being expressed a little oddly, this isn't really testing anything particularly useful — just that the popolo looks like popolo. It might be better to test that it looks like the _right_ popolo — i.e at the sha you want — by testing something that will be fairly unique at that point.